### PR TITLE
feat: Resource browser interaction fixes

### PR DESCRIPTION
**Pull request by sohel-rp**
# Description

These changes will fix multiple interactions in the resource browser,

1. Show kind name in empty states
2. Dark theme editor for the create resource modal
3. Namespace dropdown disabled with tippy
4. Search retention issue
5. Show header always,
7. Etc.

Fixes - [AB#1767](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/1767)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

All tests have been performed manually by visiting the resource browser page & perform different actions.

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx [hunk 1]
```diff
@@ -258,6 +293,7 @@ export function K8SResourceList({
         if (noResults) {
             return (
                 <ResourceListEmptyState
+                    title={`No ${selectedResource?.gvk?.Kind} found`}
```
Comment (vivek-devtron) [inline src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx:296]: We should add same default value here

File: src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx [hunk 2]
```diff
@@ -258,6 +293,7 @@ export function K8SResourceList({
...
                     subTitle={`We could not find any ${selectedResource?.gvk?.Kind}. Try selecting a different cluster${
```
Comment (vivek-devtron) [inline src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx:297]: same here

Comment (sohel-rp) [reply src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx]: Done

File: src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx (hunk 1 above)
Comment (sohel-rp) [reply src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx]: Done

**Review by shivani170 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**